packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -32,14 +31,16 @@ import 'snapshot_item.dart';
 @visibleForTesting
 enum Json {
   snapshots,
-  diffWith;
+  diffWith,
+  rootPackage;
```

Comment:
this is duplicated below. Can we store the root package once on the top level memory page json instead of duplicating amongst the sub-features?

Reply:
I tried it.
It adds big complexity in initialization flow because of multiple constructors and need of rootPackage in number of places.
I choose minor increase of json for simplicity sake.